lms/templates/header/user_dropdown.html

```diff
@@ -40,7 +41,7 @@
         <div class="mobile-nav-item dropdown-item dropdown-nav-item"><a href="${reverse('dashboard')}" role="menuitem">${_("Dashboard")}</a></div>
         <div class="mobile-nav-item dropdown-item dropdown-nav-item"><a href="${reverse('learner_profile', kwargs={'username': username})}" role="menuitem">${_("Profile")}</a></div>
         <div class="mobile-nav-item dropdown-item dropdown-nav-item"><a href="${reverse('account_settings')}" role="menuitem">${_("Account")}</a></div>
-        % if REDIRECT_TO_ORDER_HISTORY_MICROFRONTEND.is_enabled():
+        % if REDIRECT_TO_ORDER_HISTORY_MICROFRONTEND.is_enabled() and configuration_helpers.get_value('ENABLE_ORDER_HISTORY_MICROFRONTEND'):
```

Comment:
Why is this change needed?  Are we not enabling the new mfe for white-label sites?